components/layout/header.js

```diff
@@ -1,29 +1,127 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+// Libraries
+import classnames from 'classnames';
+
+// Components
 import { Link } from 'routes';
+import Icon from 'components/ui/icon';
+
+// Header components
+import DashboardHeaderContent from 'components/header-contents/dashboard/content';
+
+// Constants
+import { HEADER_MENU_LINKS } from 'constants/general';
 
 export default class Header extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false
+    };
+
+    // Bindings
+    this.onToggleMenu = this.onToggleMenu.bind(this);
+  }
+
+  onToggleMenu() {
+    this.setState({ open: !this.state.open });
+  }
+
+  getCustomContentByPage() {
+    const { url } = this.props;
+
+    switch (url.pathname) {
+      // Different pathnames for the index
+      case '/index': return { title: <h1>Kenya</h1> };
```

Comment:
I don't know and you shouldn't waste too much time on this. But we may use a regexp? look at this one ^(\/|\/index)$